### DISCUSSION TITLE
[Suggestion] Show total submissions near "Final Results" on the individual game pages

### DISCUSSION THREAD

**Discussion by zutils** [ludumdare/ludumdare#2048] [Ideas]
It would be really nice to see how our game ratings stands out from others. For example, 370th place doesn't give me much information... if there were 371 submissions, then our game was bad - if there were 3700 submissions, then our game was pretty good relative to everyone else's. Seeing a "Out of <N> submissions" lets us know how well we are doing over time. It will show us if we are looking at a rating just from Compo or just from Jam (which is confusing btw, I don't know if my 370th place is just from Jam!).

**Comment by LoneWolfHT** [comment]
Was just looking to do something like this today, best I could come up with using current info is:
```lua
x = "My ranking"
y = "Total amount of games submitted to category my game is in (Compo/Jam)"
z = "% of *all* submitted games with over 20 ratings"

result = 1 - ( x / (y*z) )
```
